common/datavalues/src/data_value.rs

```diff
@@ -275,37 +274,37 @@ impl DataValue {
 
 impl Eq for DataValue {}
 
-impl Ord for DataValue {
-    fn cmp(&self, other: &Self) -> Ordering {
+impl PartialOrd for DataValue {
```

Comment:
Is it proper to use `derive PartialOrd `  here?
~~~
#[derive(.... PartialOrd)]
pub enum DataValue {
...
}
~~~

Reply:
Got it!